python/ray/air/config.py

```diff
@@ -120,8 +122,9 @@ class ScalingConfig:
             argument.
         resources_per_worker: If specified, the resources
             defined in this Dict is reserved for each worker.
-            Define the ``"CPU"`` and ``"GPU"`` keys (case-sensitive) to
-            override the number of CPU or GPUs used by each worker.
+            Define the ``"CPU"`` key (case-sensitive) to
+            override the number of CPUs used by each worker.
+            This can also be used to request :ref:`custom resources <custom-resources>`.
```

Comment:
Does this link to Ray Core's [Custom Resources](https://docs.ray.io/en/latest/ray-core/scheduling/resources.html#custom-resources) Page?

Reply:
Ohh yeah, got it from the PR description.